### ISSUE TITLE
command not found yields exit code zero

### ISSUE THREAD

**Issue by jautz** [christiangalsterer/execbeat#15]
With execbeat version 3.1.1 on GNU/Linux exit code zero is being returned if the configured command cannot be found.

The log file contains an appropriate error message but in the produced document the exitCode suggests success:

```
  "exec": {
    "command": "FILE",
    "exitCode": 0,
    "stdout": ""
  },
```

Please consider producing [code 127](http://tldp.org/LDP/abs/html/exitcodes.html) in this case because this will help to recognise the problem's cause. Thank you.

**Comment by christiangalsterer** [comment]
Thanks for reporting, will fix it in the next days.

**Comment by christiangalsterer** [comment]
A fix is a available on master. Will try to make a release soon.

**Comment by christiangalsterer** [comment]
Released with 3.2.0
